### PR TITLE
ForkingTaskRunner: Better logging during orderly shutdown.

### PR DESCRIPTION
**Pull request by gianm**

### CODE REVIEW

File: indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java [hunk 1]
```diff
@@ -479,11 +482,16 @@ public void stop()
...
     if (timeout > 0) {
       try {
-        exec.awaitTermination(timeout, TimeUnit.MILLISECONDS);
-        log.info("Finished stopping in %,dms.", System.currentTimeMillis() - start.getMillis());
+        final boolean terminated = exec.awaitTermination(timeout, TimeUnit.MILLISECONDS);
+        final long elapsed = System.currentTimeMillis() - start.getMillis();
+        if (terminated) {
+          log.info("Finished stopping in %,dms.", elapsed);
+        } else {
+          log.warn("Executor failed to stop after %,dms, not waiting for it!", elapsed);
```
Comment (drcrallen) [inline indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java]: Should this be more alarming? like maybe Alert level?

Comment (gianm) [inline indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java]: @drcrallen sounds good, updated.

Comment (gianm) [inline indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunner.java]: also added a list of still-running tasks

**Comment by himanshug** [comment]
:+1:

**Comment by fjy** [comment]
:+1:

**Pull request merged**